### PR TITLE
Reenable Infinispan tests and disable MongoDB tests

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: src/rpkg-templates/run-tests-module-pom.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
     <artifactId>[=runTestsModule.artifactId]</artifactId>
 
-[#if runTestsModule.artifactId = 'quarkus-universe-integration-tests-camel-infinispan']
+[#if runTestsModule.artifactId = 'quarkus-universe-integration-tests-camel-mongodb']
```
Comment (ppalaga) [inline src/rpkg-templates/run-tests-module-pom.xml:13]: Hm... I am afraid, `skipTests=true` will not be enough for the mongo tests. We are actually disabling the mongo test on the Camel quarkus side https://github.com/apache/camel-quarkus/pull/1063/files#diff-32e5ef896402deb1c95ac6b572035786R94 We need to take care on the Camel Quarkus side that the mongo test won't be exported to the testJars file. Thanks for pinging me, I need to do that. Once I do it, there is no need for any special handling here (the mongo module will disappear upon re-running camel-regen).

Comment (gsmet) [reply src/rpkg-templates/run-tests-module-pom.xml:13]: Well, it works. And the idea is to reenable it later.

Comment (gsmet) [reply src/rpkg-templates/run-tests-module-pom.xml:13]: But I don't really care for the solution. I just want to be able to release the platform.

**Comment by ppalaga** [comment]
Superceded by https://github.com/quarkusio/quarkus-platform/pull/49. Please close this one.

**Comment by gsmet** [comment]
Closing as way out of date.

**Pull request closed without merging**